src/components/Article/ArticleController.tsx

```diff
@@ -185,9 +178,18 @@ const ArticleController = (props: Props) => {
     });
   };
 
+  const find = event => {
```

Comment:
rename find method to handleSearchCriteriaChange. And just set the searchCriteria value. Use useEffect to do the actual filter

Reply:
Changes done as per the comments